src/apps/schema/src/appRevamp/components/utils.ts

```diff
@@ -0,0 +1,87 @@
+import { replace, isEmpty } from "lodash";
```

Comment:
This file was recently moved to `utils/index.ts` by @agalin920. This file that you've added should no longer exist.

Reply:
Got it, removed in my latest commit.